get_hashsets.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
+
+const AppConstants = require("./app-constants").init();
+
+const fs = require("fs");
+const path = require("path");
+const request = require("request");
+
+const pkg = require("./package.json");
+
+const HIBP_AUTH = `Bearer ${AppConstants.HIBP_API_TOKEN}`;
+const USER_AGENT = `${pkg.name}@${pkg.version}`;
+const BREACH_HASHSET_DIR = "breach_hashsets";
+
+
+function getBreachHashset(breach) {
+  if (breach.IsActive && breach.IsVerified && breach.DataClasses.includes("Email addresses")) {
+    console.log("Active, verified breach with email addresses: " + breach.Name);
+    const url = AppConstants.HIBP_API_ROOT + "/enterprisesubscriber/hashset/" + breach.Name;
```

Comment:
WARNING: begin personal opinion rant that is not important for making progress on this repo:

I personally hold the opinion that const provides no real value and only adds decision pressure while writing code. It's also (imo) confusing, since it only prevents reassignment - which means you need to be very aware of exactly what operators count as reassignment in JS to really understand the contract that const enforces. A great post about this: http://web.archive.org/web/20160315114714/http://blog.getify.com/constantly-confusing-const/. But if you're comfortable with consts, I won't complain further.

Reply:
`const` is at least [a signal-of-intent by the developer that identifier won't/shouldn't be re-assigned](https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75). I see that only makes the value immutable when assigning primitives, but meh - Kyle's post reads more like one JS expert's opinion, not a technical principle. 

I'm okay either way - whatever we want, let's just make it part of an `eslint` config so we're consistent across the code-base.